web/app/Application.js

```diff
@@ -104,6 +104,10 @@ Ext.define('Traccar.Application', {
         return window.matchMedia && window.matchMedia('(max-width: 768px)').matches;
     },
 
+    isVehicleFeaturesDisabled: function () {
```

Comment:
Name sounds a bit weird because "is" and plural. Maybe `getVehicleFeaturesDisabled`?